packages/hoppscotch-common/src/components/app/Header.vue

```diff
@@ -281,8 +282,19 @@ const showTeamsModal = ref(false)
 const breakpoints = useBreakpoints(breakpointsTailwind)
 const mdAndLarger = breakpoints.greater("md")
 
+const { content: banner } = useService(BannerService)
 const network = reactive(useNetwork())
 
+watch(network, () => {
+  if (!network.isOnline) {
+    banner.value = {
+      type: "info",
+      text: t("helpers.offline"),
+      alternateText: t("helpers.offline_short"),
+    }
+  } else banner.value = undefined
```

Comment:
```suggestion
  if (network.isOnline) {
    banner.value = undefined;
    return;
  }

  banner.value = {
    type: "info",
    text: t("helpers.offline"),
    alternateText: t("helpers.offline_short"),
  };
```

How about the early return approach?